run.py

```diff
@@ -42,7 +42,7 @@
     'reconfigEnabled': 'false',
     'dataDir': ZOOKEEPER_DATA_DIR,
     'quorumListenOnAllIPs': True,
-    'clientPort': get_port('client'),
+    'clientPort': get_port('client', 2181),
```

Comment:
Looks like we would fail in build_node_repr too on line 64

Reply:
Indeed. It did not fail in my setup since I run just one node, so `build_node_repr` does not get called. Added the default port there too.